### DISCUSSION TITLE
Fokus von Inputs innerhalb einer Tabelle

### DISCUSSION THREAD

**Discussion by VladF146** [public-ui/kolibri#6148] [Q&A]
Ich soll eine Tabelle implementieren, die in der ersten Zeile pro Spalte ein Inputfeld zur Filterung hat. Die Inputs bekomme ich mit getRoot(el).render() in die Tabelle rein.
Das Problem entsteht, wenn ich mit den Inputs den local State oberhalb der Tabelle anpasse und dadurch die komplette Tabelle samt Inputs rerendere. Die Inputs verlieren so den Fokus.

Ich könnte den Fokus manuell setzen, komme aber ohne etliche Kombinationen von querySelector()?.shadowRoot (bad practice) nicht an focus().

Gibt es eine best practice für einen solchen Fall, außer die Inputs oberhalb der Tabelle anzusiedeln? 😄

**Comment by deleonio** [comment]
@cbraehmig Bitte beantworten.

Wir sehen hier ein alternatives Bedienkonzept. Kurze Rückfrage @VladF146 ... ist das ein ITZBund-Projekt?
